src/server/visual-diff-plugin.js

```diff
@@ -114,56 +139,100 @@ export function visualDiff({ updateGoldens = false, runSubset = false } = {}) {
 			const newPath = join(rootDir, PATHS.VDIFF_ROOT, testPath, dir);
 			const goldenFileName = `${join(newPath, PATHS.GOLDEN, browser, newName)}.png`;
 			const passFileName = `${join(newPath, PATHS.PASS, browser, newName)}.png`;
-			const screenshotFileName = `${join(newPath, PATHS.FAIL, browser, newName)}.png`;
-
-			if (!isCI) { // CI will be a fresh .vdiff folder each time and only one run
-				if (session.testRun !== currentRun) {
-					currentRun = session.testRun;
-					clearedDirs.clear();
-				}
+			const screenshotFile = join(newPath, PATHS.FAIL, browser, newName);
+			const screenshotFileName = `${screenshotFile}.png`;
+
+			if (command === 'brightspace-visual-diff-compare') {
+				if (!isCI) { // CI will be a fresh .vdiff folder each time and only one run
```

Comment:
I thought about putting this whole cleanup in it's own command to make this cleaner, but then we're running https://github.com/BrightspaceUI/testing/blob/97239155d8cb3a3df32707eea35059c0726d3c13/src/server/visual-diff-plugin.js#L136-L143 3 times per test.  Thoughts?

Reply:
Yeah... was thinking we could also refactor this a bit at this point to be like:

```javascript
if (command === 'brightspace-visual-diff-compare') {
  doCompare();
} else if (command === 'brightspace-visual-diff-compare-resize') {
  doCompareResize();
}
```

At the very least that'd reduce a lot of the nesting, and would also let us move things out into separate files if that made sense.

Reply:
Yeah I started doing that and then there were way too many things I was passing around as parameters without knowing what you'd need and where, so I figured that might be better once the skeleton of the report stuff is in